### ISSUE TITLE
Any whitespace character in URI list file should be treated as separator

### ISSUE THREAD

**Issue by dylanjboyd** [jrnewell/spotify-ripper#24]
# Whitespace separator
Currently, URIs in an input text file must be separated by new lines. To make the application more flexible, and to safeguard against errors caused by unexpected input, `spotify-ripper` could treat **any whitespace character** as a separator of URIs.

```
spotify:album:3sbscaAkfdUlI2DdThg4Vc
spotify:album:0INXDk2mRAWTRMs2Mo1pXZ spotify:album:13LmHHsOBfEt0siKMOypjf
spotify:album:5Ph65zqUzVnIPLJCySmQ5T spotify:album:5LhbwhBfshZF4MaTCx5kyr spotify:user:spotifyaustralia:playlist:4Kz5zSgyZMI1tK2n4RlIkb spotify:album:6j4PKBoTlHPweQNSGlu9pX
```

# Invalid character separator
Furthermore, the separator character could be anything which isn't alphanumeric, or `:`, since Spotify URIs themselves are limited to these characters. If this were the case, the following could be provided as an input file:

```
spotify:album:3sbscaAkfdUlI2DdThg4Vc
spotify:album:0INXDk2mRAWTRMs2Mo1pXZ spotify:album:13LmHHsOBfEt0siKMOypjf,spotify:album:5Ph65zqUzVnIPLJCySmQ5T-spotify:album:5LhbwhBfshZF4MaTCx5kyr+spotify:user:spotifyaustralia:playlist:4Kz5zSgyZMI1tK2n4RlIkb#$%^spotify:album:6j4PKBoTlHPweQNSGlu9pX
```

# Perceived input
```
spotify:album:3sbscaAkfdUlI2DdThg4Vc
spotify:album:0INXDk2mRAWTRMs2Mo1pXZ
spotify:album:13LmHHsOBfEt0siKMOypjf
spotify:album:5Ph65zqUzVnIPLJCySmQ5T
spotify:album:5LhbwhBfshZF4MaTCx5kyr
spotify:user:spotifyaustralia:playlist:4Kz5zSgyZMI1tK2n4RlIkb
spotify:album:6j4PKBoTlHPweQNSGlu9pX
```

**Comment by jrnewell** [comment]
I committed a fix for this in e404b09 / version ```1.2.1```.

**Comment by jrnewell** [comment]
There was a bug with this change.  Fixed in version ``1.2.3``.
